### PR TITLE
Fixed logging again

### PR DESCRIPTION
**Pull request by mssola**

### CODE REVIEW

File: images_test.go [hunk 1]
```diff
@@ -117,6 +89,8 @@ func TestImagesListOk(t *testing.T) {
 	}
 }
 
+// Special tests for the IMAGES command.
+
```
Comment (flavio) [inline images_test.go]: Why the empty line? Not an issue of course

**Comment by flavio** [comment]
The code looks good :clap: , it also fixed most of the issues we had inside of the integration tests. However there are still 2 integration tests that are failing.

If you want you can fix them with another PR, I'll leave the decision to you.

File: images_test.go (hunk 1 above)
Comment (mssola) [inline images_test.go]: These comments are there to group a set of test functions, or to divide a test file between different (but related) commands. In the case you pointed out, it serves for grouping two test functions.

**Comment by mssola** [comment]
@flavio I'll fix these test failures now.

**Comment by mssola** [comment]
Don't merge yet, I want to squash all the commits after tests are green again ;)

**Comment by mssola** [comment]
Now it's mergeable :)

**Pull request merged**